library/src/morpheus-master/sass/modules/navigation/_base.scss

```diff
@@ -21,7 +21,6 @@ body.is-logged-out{
 
 	.#{$namespace}headerLogo{
 		width: 100%;
-		overflow: hidden;
```

Comment:
@fostersdesign do you think this overflow hidden is useless? I needed to remove it to show the "Quick Links" box after I've moved `<ul class="Mrphs-quickLinksNav__subnav">` element into `<li class="Mrphs-quickLinksNav__popup">`

Reply:
I think the "overflow: hidden" is less useful now that we've flexed most of the containers. So, yes, I think it should be okay to remove it.